### PR TITLE
fix: fixed cropped destination field in send payment flow

### PR DESCRIPTION
**Pull request by Aditi-Singh16**
Fixes issue #2206 
**Preview**
image
image

### CODE REVIEW

**Comment by Aditi-Singh16** [comment]
**Update**
image

File: app/components/payment-destination-display/payment-destination-display.tsx [hunk 1]
```diff
@@ -40,14 +43,12 @@ export const PaymentDestinationDisplay = ({
 
   // we assume this is a bitcoin address or lightning invoice
   // not a username
-  const firstSix = destination.slice(0, 5)
-  const lastSix = destination.slice(-5)
-  const middle = destination.slice(5, -5)
+  const firstSix = destination.slice(0, -6)
```
Comment (nicolasburtey) [inline app/components/payment-destination-display/payment-destination-display.tsx:46]: that doesn't sounds right. this select all the last 6 caracters. so the first 6 are no longer in bold.

**Review by nicolasburtey — CHANGES_REQUESTED** [review]
need to fix the bold first 6 caracters

File: app/components/payment-destination-display/payment-destination-display.tsx (hunk 1 above)
Comment (Aditi-Singh16) [reply app/components/payment-destination-display/payment-destination-display.tsx]: image

**Comment by nicolasburtey** [comment]
Can you show a new screenshot update?

**Comment by Aditi-Singh16** [comment]
> Can you show a new screenshot update?

Yes, this is the one - https://github.com/GaloyMoney/galoy-mobile/pull/2211#discussion_r1203898350

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**